sunpy/visualization/animator/wcs.py

```diff
@@ -110,7 +103,10 @@ def _get_wcs_labels(self):
         Read first the axes names property of the wcs and fall back to physical types.
         """
         # world_axis_names was only added to the APE 14 API in 4.0, so do this for backwards compatibility.
-        world_axis_names = getattr(self.wcs, "world_axis_names", [''] * self.wcs.world_n_dim)
+        try:
+            world_axis_names = self.wcs.world_axis_names
+        except AttributeError:
+            world_axis_names = [''] * self.wcs.world_n_dim
```

Comment:
I am not sure we need this.

Reply:
@Cadair said that this method can be rewritten not to use `getattr`, but I could have misunderstood the way to do it